### PR TITLE
base ui

### PR DESCRIPTION
**Pull request by juanii000**
First commit with base UI 

image

### CODE REVIEW

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="ToolbarStyle" parent="Theme.AppCompat.NoActionBar">
+        <item name="android:background">@color/teal_700</item>
+        <item name="android:title">@string/app_name</item>
+        <item name="android:titleTextColor">@color/white</item>
+    </style>
+    
+    <style name="button_main_page">
```
Comment (BQuinteros) [inline app/src/main/res/values/styles.xml]: the name of this styles is in cammel case like ToolbarStyle maybe this can be named ButtonMainPageStyle

**Comment by BQuinteros** [comment]
pls try to avoid this in the pull request, the solution is doing an empty space at end of the file
image

File: app/src/main/res/values/styles.xml (hunk 1 above)
Comment (juanii000) [reply app/src/main/res/values/styles.xml]: Perfect, I have a mess on my brain :P

**Review by SartiMau — CHANGES_REQUESTED** [review]
pls transform the jpg file to a webp file

**Review by SartiMau — APPROVED** [review]

**Review by lucianoeli — APPROVED** [review]

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -4,15 +4,21 @@
...
-        android:text="Hello World!"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+    <Button
+        android:id="@+id/button_parking_size"
```
Comment (nicolbidone) [inline app/src/main/res/layout/activity_main.xml]: please use the suggested naming convention
https://jeroenmols.com/blog/2016/03/07/resourcenaming/

Comment (juanii000) [reply app/src/main/res/layout/activity_main.xml]: Done, thanks !

**Review by BQuinteros — APPROVED** [review]

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (mauroLonghi) [reply app/src/main/res/layout/activity_main.xml]: pls, use the suggested convention given in the on bording course

**Review by mauroLonghi — APPROVED** [review]

**Review by nicolbidone — APPROVED** [review]

**Pull request merged**